project-templates/java-emf-theia/glsp-server/src/main/java/org/eclipse/glsp/example/javaemf/TaskListDiagramConfiguration.java

```diff
@@ -0,0 +1,46 @@
+/********************************************************************************
+ * Copyright (c) 2022 EclipseSource and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * https://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+package org.eclipse.glsp.example.javaemf;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+import org.eclipse.emf.ecore.EClass;
+import org.eclipse.glsp.example.javaemf.util.TaskListModelTypes;
+import org.eclipse.glsp.graph.DefaultTypes;
+import org.eclipse.glsp.server.diagram.BaseDiagramConfiguration;
+import org.eclipse.glsp.server.types.EdgeTypeHint;
+import org.eclipse.glsp.server.types.ShapeTypeHint;
+
+public class TaskListDiagramConfiguration extends BaseDiagramConfiguration {
+
+   @Override
+   public Map<String, EClass> getTypeMappings() { return DefaultTypes.getDefaultTypeMappings(); }
+
+   @Override
+   public List<ShapeTypeHint> getShapeTypeHints() {
+      return List.of(new ShapeTypeHint(TaskListModelTypes.TASK, false, true, false, false));
```

Comment:
I think it would be great if the node would be movable and resizable.

Reply:
Yes I agree, I added that with the current commit